### PR TITLE
Specify npm required version

### PR DESCRIPTION
**Pull request by thisismydesign**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ It has
 - [~~npmignore~~ npm files](https://github.com/c-hive/guides/blob/2d3e41b73515a571dd78576e02d2fd15fe786d9e/js/package.md#what-to-include)
 - [dependency comments](https://github.com/c-hive/guides/blob/2d3e41b73515a571dd78576e02d2fd15fe786d9e/js/best-practices.md#comment-dependencies-in-the-packagejson)
 - dependency security fixes
+- npm version requirement [to avoid `package-lock` ping-pong](https://github.com/c-hive/guides/blob/24ce093e59f89374ed2bf0a2e1fc5c45e5490044/js/best-practices.md#specify-npm-version-requirements-to-avoid-package-lock-ping-pong)
```
Comment (gomorizsolt) [inline README.md]: Thanks for reading up on this topic! As a side note, there's a comment where somebody describes specifying the 6.6+ version doesn't solve the issue. We'll see later.

**Review by gomorizsolt — APPROVED** [review]
Thanks for reading up on this topic! As a side note, there's a comment where somebody describes specifying the 6.6+ version doesn't solve the issue. We'll see later.

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+engine-strict=true
```
Comment (gomorizsolt) [inline .npmrc:1]: One question though, what's this statement supposed to do?

Comment (gomorizsolt) [reply .npmrc:1]: Nevermind.

> If set to true, then npm will stubbornly refuse to install (or even consider installing) any package that claims to not be compatible with the current Node.js version.

Comment (thisismydesign) [reply .npmrc:1]: It's actually something else for our case. Otherwise `npm install` won't error with the wrong npm version.

**Comment by thisismydesign** [comment]
> there's a comment where somebody describes specifying the 6.6+ version doesn't solve the issue. We'll see later.

Yep. Not very useful though as the commenter didn't specify the issue. So yes, we'll see.

**Pull request merged**